ssl/statem/extensions_clnt.c

```diff
@@ -498,6 +495,35 @@ int tls_construct_ctos_supported_versions(SSL *s, WPACKET *pkt, X509 *x,
     return 1;
 }
 
+/*
+ * Construct a psk_kex_modes extension. We only have two modes we know about
+ * at this stage, so we send both.
+ */
+int tls_construct_ctos_psk_kex_modes(SSL *s, WPACKET *pkt, X509 *x,
```

Comment:
I am fairly sure we have a gazillion instances of this throughout the code, but there only seems to be two uses of osslargused. I think some compilers can emit warnings on this...but its probably a whole other PR to look at that (having first had the debate about whether we care). Probably doesn't make sense to do something with this one isolated instance.